charts/core/traefik/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog<br>
 
 
+<a name="traefik-11.0.14"></a>
+### [traefik-11.0.14](https://github.com/truecharts/apps/compare/traefik-11.0.13...traefik-11.0.14) (2022-04-04)
+
+#### Feature
+
+* Add `stripPrefixRegex` Traefik middleware ([#2422](https://github.com/truecharts/apps/pull/2422), [documentation](https://doc.traefik.io/traefik/middlewares/http/stripprefixregex/))
+
+
+
```

Comment:
NEVER touch the changelog.
EVER.

All md files are autogenerated by ci touching them breaks things.